src/ui/components/layouts/base_layout/side_nav.rs

```diff
@@ -11,22 +11,22 @@ pub fn SideNav() -> impl IntoView {
   let i18n = use_i18n();
 
   view! {
-    <nav aria-label="Pages nav">
+    <section aria-label="Main navigation links">
```

Comment:
`section`s seem a better semantic choice for groups of links within a nav than several separate navs.

Reply:
Cool, the actual html tags matter to me less than just making sure the components are modular.